src/utils/consumptionHelpers.ts

```diff
@@ -0,0 +1,37 @@
+export function getAverage(data: number[]): number {
+    try {
```

Comment:
not sure why do you wrap eerything with try catch, kinda takes a bit of room, don't you have validation for inputs?